src/app/client/src/app/app.component.html

```diff
@@ -71,6 +71,8 @@
       [userFeedData]="userFeed"> </app-validate-teacher-identifier-popup>
     <app-user-onboarding (close)="onLocationSubmit()" *ngIf="showUserTypePopup || !isLocationConfirmed"
       [isCustodianOrgUser]="isCustodianOrgUser" [deviceProfile]="deviceProfile"></app-user-onboarding>
+    
+    <app-year-of-birth *ngIf="isCustodianOrgUser && usersProfile && !usersProfile?.dob && (usersProfile?.userLocations?.length>0 || showYearOfBirthPopup)"></app-year-of-birth>
```

Comment:
is this only needed for CustodianOrgUser not the state users?

Reply:
yes, this is need for Custodian users only because,
Assumption : when a user is signing up using SSO , they will be always above 18